typing_extensions/src_py3/test_typing_extensions.py

```diff
@@ -1552,11 +1552,15 @@ def test_total(self):
         self.assertEqual(D(), {})
         self.assertEqual(D(x=1), {'x': 1})
         self.assertEqual(D.__total__, False)
+        self.assertEqual(D.__required_keys__, frozenset())
+        self.assertEqual(D.__optional_keys__, {'x'})
 
         if PY36:
             self.assertEqual(Options(), {})
             self.assertEqual(Options(log_level=2), {'log_level': 2})
             self.assertEqual(Options.__total__, False)
+            self.assertEqual(Options.__required_keys__, frozenset())
```

Comment:
I don't know whether this test and the one below should be within the `if PY36` block or not.

Reply:
`Options` only exists if PY36 is on (it's defined with variable annotations), so these tests should be within the `if PY36` block.